appscale/tools/local_state.py

```diff
@@ -965,15 +980,15 @@ def generate_crash_log(cls, exception, stacktrace):
 
 
   @classmethod
-  def ensure_user_wants_to_terminate(cls):
-    """ Asks the user for confirmation before we terminate their AppScale
+  def ensure_user_wants_to_clean(cls):
+    """ Asks the user for confirmation before we clean their AppScale
     deployment.
 
     Raises:
       AppScaleException: If the user does not want to terminate their
         AppScale deployment.
     """
-    cls.confirm_or_abort("Terminating AppScale will delete all stored data.")
+    cls.confirm_or_abort("Cleaning or terminating AppScale will delete all stored data.")
```

Comment:
Would you prefer to have 2 functions then with 2 different messages? Or should we phrase is differently, say The following operation will delete or may delete ...?